lib/cfa/augeas_parser.rb

```diff
@@ -35,6 +35,47 @@ module CFA
   class AugeasElement < Hash
   end
 
+  # Error that is raised when augeas fail for some reason.
+  # Base class for more specialized exceptions.
+  class AugeasError < RuntimeError
```

Comment:
IMHO this base class is not useful, because it mixes AugeasInternalError (500 Internal error, send us logs) with AugeasSerializingError (400 Client error, fix your data).